### DISCUSSION TITLE
GitHub Support does not respond even after 1 month

### DISCUSSION THREAD

**Discussion by deepacekarki** [community/community#129809] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My other account was compromised about a month ago, and the email address was changed. I created a ticket for it, but I haven't received any response yet. I'm unsure if I'll ever get my account back, but I would appreciate it if someone could look into it.

Ticket ID: 2812881  
Username: dipeshkarki005

**Comment by ebndev** [comment]
Hi @deepacekarki,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
